.github/workflows/cmake.yml

```diff
@@ -20,13 +20,13 @@ jobs:
           compiler_prefix: /usr/bin
         - compiler_driver: clang++
           compiler_prefix: /usr/bin
-          cxx_flags_extra: "-stdlib=libc++"
+          cxx_flags_extra: "-stdlib=libc++ -Wno-c++17-attribute-extensions -Wno-gnu-zero-variadic-macro-arguments"
         - compiler_driver: icpx
           compiler_prefix: /opt/intel/oneapi/compiler/latest/linux/bin
           # To get new URL, look here:
           # https://www.intel.com/content/www/us/en/developer/articles/tool/oneapi-standalone-components.html#inpage-nav-6-undefined
           compiler_url: https://registrationcenter-download.intel.com/akdlm/IRC_NAS/ebf5d9aa-17a7-46a4-b5df-ace004227c0e/l_dpcpp-cpp-compiler_p_2023.2.1.8_offline.sh
-          cxx_flags_extra: "-DMDSPAN_USE_BRACKET_OPERATOR=0"
+          cxx_flags_extra: "-DMDSPAN_USE_BRACKET_OPERATOR=0 -Wno-c++17-attribute-extensions -Wno-gnu-zero-variadic-macro-arguments"
         - enable_benchmark: ON
```

Comment:
Won't switching to Debug affect the benchmark times? Maybe we should add build type to the matrix

Reply:
Are these benchmark times recorded anywhere? To me, it seems we really only test correctness in the CI.